main

```diff
@@ -0,0 +1,347 @@
+//lists of objects
```

Comment:
I pretty much just raw copy and paste the code to a text file and upload it to github so it might cause some formatting issue. Also, explain why this file doesn't have .pde extension at the end. In order to run the code, all you need to do is to copy and paste all the text and run it on processing.